README.md

```diff
@@ -46,14 +46,27 @@ If I plan on doing a fair amount of custom development as part of a project, I l
 
 If you aren't planning on doing much custom development (e.g. maybe you are just installing already existing modules and extensions) it might make more sense to take advantage of the CMS and CiviCRM already installed on the image (at `/var/www/html`).
 
-To get a local project up and running, choose your CMS flavour from the [docker-compose](docker-compose) and copy the entire directory to somewhere like `~/projects/backdrop-example-project`
+### Build
 
-1. Copy the `dev.dist.env` file to `.env`. and edit as appropriate (choose sensible passwords, set the base url and other required environment variables, e.g. the domain you want the site to be available at.
-2. Copy `docker-compose.dev.dist.yml` to `docker-compose.yml` and assign port numbers as appropriate (see _Reverse proxies_ for more details).
-3. _If you are mounting a mono repo from the host to the container_ (see above), place it in the `src` folder, and uncomment the `./src:/var/www/html` volume.
-4. _If you want development site to be available at a memorable address and/or you want to serve it via https_, configure it now.
-5. Run `docker-compose up -d`
-6. Install a new site with something like `docker-compose exec civicrm civicrm-docker-install` .
+This repo will allow you to locally create the images you need. The process works with a minimal Python3 environment.
+You may wish to create a virtual environment and install the dependencies in `requirements-dev.txt`.
+
+1. If you don't want to build all the possible image combinations yourself, tweak `variables.py` to contain only the environments you need.
+1. Run `./generate.py`, it will update the `combos.json` file with only the combinations setup at `variables.py`.
+1. Run `./build.py`, it will create and tag the images as specified by `combos.json`.\
+If working with Wordpress, make sure you note down the `civicrm` user UID and GID from the last step.
+
+### Run images
+
+To get a local project up and running, choose your CMS flavour from the [docker-compose folder](docker-compose) and copy the entire directory to somewhere like `~/projects/backdrop-example-project`.
+
+1. In the new folder, move `dev.dist.env` to `.env`. and edit as appropriate (choose sensible passwords, set the base url and other required environment variables, e.g. the domain you want the site to be available at).
+1. In the new folder, move `docker-compose.dev.dist.yml` to `docker-compose.yml` and assign NNNN port numbers as appropriate (see _Reverse proxies_ for further tweaks).\s
```

Comment:
we are now using traefik as a reverse proxy internally. It works nicely. We don't want to force this on anyone. It would be nice to update the examples to make them reverse proxy / deployment agnostic.